src/router/graphql.ts

```diff
@@ -55,6 +63,12 @@ if (config.graphql) {
       // This will throw an error if the user is not allowed to access the specified DB
       (await metadata.getDb(dbMan.getUid(req.user), req.params.db)).pgName :
       req.params.db;
-    getPostgraphile(db)(req, res, next);
+    try {
+      getPostgraphile(db)(req, res, next);
+    } catch (e) {
+      // This path is if postgraphile successfully spun up but somehow crapped out anyway
+      logger.error('Postgraphile failure', { e, });
+      res.status(400).end(`${db} does not exist`);
+    }
```

Comment:
This path is not hit, afaict, *but* it still seems valuable to have in case we ever do get into this situation.